content/docs/14-troubleshooting/01-orchestration-spectrocloud.md

```diff
@@ -14,38 +14,66 @@ import PointsOfInterest from 'shared/components/common/PointOfInterest';
 
+
+<br />
+
+<br />
+
 1. New K8S cluster request from user submitted to the cluster management system.
 
-2. Spectro Cloud creates the Cluster-API (CAPI) custom-resource specifications for the target cloud, e.g in VMware this would translate to: Cluster, vSphereCluster, KubeadmControlPlane (with replica 1), and VSphereMachineTemplate. These resources are created in the management cluster.
 
-3. Cluster API and corresponding cluster-api provider, e.g: cluster-api-provider-vsphere, provisions the first control-plane node CP-A on the target cloud. 
+2. Palette creates the Cluster-API (CAPI) custom-resource specifications for the target cloud, e.g in VMware this would translate to: Cluster, vSphereCluster, KubeadmControlPlane (with replica 1), and VSphereMachineTemplate. These resources are created in the management cluster.
 
-4. When CP-A is operational, the management platform will install a Spectro Cloud agent into the workload cluster and then perform a pivot of the Cluster API resources. 
 
-5. CP-A agent will retrieve the latest specifications for the cluster, e.g: 3 control-plane, and 3 workers. CP-A will generate and update the remaining CAPI resources, e.g: update replicas to 3 for KubeadmControlPlane, create the worker's MachineDeployment and VSphereMachineTemplate, etc.  Cluster API running in CP-A will provision the remaining control plane and worker nodes.
+3. Cluster API and corresponding cluster-api provider, e.g: cluster-api-provider-vsphere, provisions the first control-plane node CP-A on the target cloud.
 
-6. The Spectro Cloud Agent will install all the additional add-ons as specified by the cluster’s cluster profile (e.g: logging, monitoring, security).
 
+4. When CP-A is operational, the management platform will install a Palette agent into the workload cluster and then perform a pivot of the Cluster API resources.
+
+
+5. CP-A agent will retrieve the latest specifications for the cluster, e.g: 3 control-plane, and 3 workers. CP-A will generate and update the remaining CAPI resources, e.g: update replicas to 3 for KubeadmControlPlane, create the worker's MachineDeployment or VSphereMachineTemplate. Cluster API running in CP-A will provision the remaining control plane and worker nodes.
+
+
+6. The Palette agent will install all the additional add-ons as specified by the cluster's cluster profile (e.g: logging, monitoring, security).
 
- 
-## Why Spectro Cloud Pivots
 
-As part of the workload K8s cluster provisioning, only the first control-plane node is launched by Cluster API, running in the Spectro Cloud management cluster. Once the control-plane node is operational, Cluster API resources are _pivoted_ from the management platform into the target workload cluster.
+## Why Palette Pivots
+
+<br />
+
+Palette's decentralized model is based on a "decentralized management - local policy enforcement" scalable architecture.
+
+![distributed_orchestration.png](distributed_orchestration.png)
+
+
```

Comment:
<img width="993" alt="image" src="https://user-images.githubusercontent.com/49595451/177168071-9e08a745-e5a0-4d93-8520-6079f6276c29.png">
* in the image plz note the red marks. 
* 2 dots and also internal components circled is not visible on zooming.

Reply:
removed the artifacts and darkened text